src/loki/status_action.cc

```diff
@@ -64,6 +68,10 @@ void loki_worker_t::status(Api& request) const {
   status->set_has_admins(tile && tile->header()->admincount() > 0);
   status->set_has_timezones(tile && tile->node(0)->timezone() > 0);
   status->set_has_live_traffic(reader->HasLiveTraffic());
+
+  status->set_has_transit_tiles(filesystem::exists(transit_dir) &&
+                                filesystem::is_directory(transit_dir));
+  status->set_osm_changeset(tile && tile->header()->dataset_id());
```

Comment:
either connectivity_map or get_tileset and cache the result. problem is for tarred data